src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -277,6 +278,10 @@ private async Task CheckLoginRequestStatus()
                     await HandleLoginCompleteAsync();
                 }
             }
+            catch (ApiException ex) when (ex?.Error?.StatusCode == System.Net.HttpStatusCode.BadRequest)
+            {
+                HandleException(ex, ex?.Error?.Message);
+            }
```

Comment:
⛏️ `ex` is never null and inside given that you've already checked `null` in the `when` condition, is not necessary again.
```suggestion
            catch (ApiException ex) when (ex.Error?.StatusCode == System.Net.HttpStatusCode.BadRequest)
            {
                HandleException(ex, ex.Error.Message);
            }
```

Reply:
🤔 @andrebispo5  Can't remember the reason, why are we not using the base `HandleException`? which already takes care of this.

Reply:
Didn't noticed this was using a private HandleException class, this must have been before we had the base method done. I'll just remove the private and let it use the base one.